src/utils/logic.js

```diff
@@ -1,6 +1,5 @@
 import { ZIPCODES } from "../data/zipcodes";
-import { navigateTo } from "gatsby-link";
-import { array } from "lodash";
```

Comment:
LOL so this `import { array } from "lodash"` was actually importing `_` into the global namespace with _only_ the array functions exposed.  Which is really dumb.  This imports all of lodash as `_` which does potentially bloat our JS bundle but also makes things way more readable.